templates/tab/js/m365/README.md

```diff
@@ -10,18 +10,20 @@ Microsoft 365 supports the ability to run web-based UI inside "custom tabs" that
 
 - [NodeJS](https://nodejs.org/en/)
 - An M365 account. If you do not have M365 account, apply one from [M365 developer program](https://developer.microsoft.com/en-us/microsoft-365/dev-program)
-- [Teams Toolkit Visual Studio Code Extension](https://aka.ms/teams-toolkit) version after 1.55 or [TeamsFx CLI](https://aka.ms/teamsfx-cli)
+- [Teams Toolkit Visual Studio Code Extension](https://aka.ms/teams-toolkit) version after 3.8.0 or [TeamsFx CLI](https://aka.ms/teamsfx-cli)
 
 ## Debug
 ### From Visual Studio Code
-- Use the `Run and Debug Activity Panel` in Visual Studio Code, select `Debug in Teams`, `Debug in Outlook` or `Debug in Office` and click the `Run and Debug` green arrow button.
-- If you select `Debug in Outlook` or `Debug in Office`, during debugging, a VS Code dialog will be popped up as the image below. Please click "Install in Teams" first to install the app in Teams, then click "Continue" to continue debugging the app in Outlook or Office.
-![Install in Teams](./images/install-in-teams-vsc.png)
+1. Use the `Run and Debug Activity Panel` in Visual Studio Code, select `Debug in Teams`, `Debug in Outlook` or `Debug in Office` and click the `Run and Debug` green arrow button.
+2. If you select `Debug in Outlook` or `Debug in Office`, during debugging, a VS Code dialog will be popped up as the image below. Please click "Install in Teams" first to install the app in Teams, then click "Continue" to continue to debug the app in Outlook or Office.
+
+  ![Install in Teams VSC Local](./images/install-in-teams-vsc-local.png)
 
 ### From TeamsFx CLI
-- Start debugging the project by executing the command `teamsfx preview --local --m365-host <m365-host>` in your project directory, where `m365-host` is `teams`, `outlook` or `office`.
-- If you select `m365-host` as `outlook` or `office`, during debugging, a dialog will be popped up as the image below. Please select "Install in Teams" first to install the app in Teams, then select "Continue" to continue debugging the app in Outlook or Office.
-![Install in Teams](./images/install-in-teams-cli.png)
+1. Start debugging the project by executing the command `teamsfx preview --local --m365-host <m365-host>` in your project directory, where `m365-host` is `teams`, `outlook` or `office`.
+2. If you select `m365-host` as `outlook` or `office`, during debugging, a dialog will be popped up as the image below. Please select "Install in Teams" first to install the app in Teams, then select "Continue" to continue to debug the app in Outlook or Office.
+
+  ![Install in Teams CLI](./images/install-in-teams-cli.png)
 
 ## Edit the manifest
 
```

Comment:
`The folder contains two manifest files` -> `The folder contains a manifest template file`